zap/src/main/java/org/zaproxy/zap/extension/httppanel/HttpPanel.java

```diff
@@ -238,10 +238,10 @@ public void setMessage(Message msg) {
         Message oldMessage = this.message;
         this.message = msg;
 
-        for (Iterator<Entry<String, HttpPanelComponentInterface>> it =
-                        components.entrySet().iterator();
-                it.hasNext(); ) {
-            HttpPanelComponentInterface component = it.next().getValue();
+        for (Entry<String, HttpPanelComponentInterface> stringHttpPanelComponentInterfaceEntry :
```

Comment:
Any reason why the name is so verbose? Could be just `entry`, it could also be changed to just iterate the values.

Reply:
Must be how the IDE autogenerated it and yes, I agree on the values thing.. amendment coming right up